docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,601 @@
 ---
-title: Teleport Trusted Clusters
-description: How to configure access and trust between two Teleport Clusters.
-h1: Trusted Clusters
+title: Configure Trusted Clusters
+description: Explains how you can configure a trust relationship and manage access between two Teleport clusters.
+h1: Configure trusted clusters
 ---
 
-Teleport can partition compute infrastructure into multiple clusters. A cluster
-is a group of resources connected to a Teleport Auth Server. Each cluster
-manages a separate certificate authority (CA) for all users and resources.
+As you learned in [Core Concepts](../../core-concepts.mdx), a Teleport cluster
+consists of the Teleport Auth Service, the Teleport Proxy Service, and the 
+Teleport services that manage access to resources in your infrastructure.
+With Teleport, you can also partition your infrastructure into multiple connected 
+clusters to allow the users of one cluster—the **root cluster**—to connect to resources 
+in another cluster—the **leaf cluster**—while authenticated using a single 
+Teleport Auth Service. 
 
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly connect to resources in another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-run behind a firewall without any ingress ports open.
+After you establish a trust relationship between the root and leaf clusters, the leaf cluster can
+run behind a firewall without any inbound ports open. The leaf cluster creates an outbound reverse 
+SSH tunnel to the root cluster and keeps the tunnel open.
+When a user tries to connect to any resource in the leaf cluster, the connection goes through the 
+reverse tunnel using the root cluster's proxy.
```

Comment:
Does "proxy" mean "Proxy Service" here, or another proxy?

Reply:
Based on the surrounding context the references to proxy mean the "Teleport Proxy Service instance running on the root cluster." I still think the overuse of capitalization to emphasize productization is more distracting than helpful. Plus, it seems to run counter to the messaging issue:
18:1-20:44  warning  Incorrect messaging: "Desktop Access " (in body text). Avoid the suggestion that Teleport includes multiple products, rather than a unified access platform. You should write about adding a resource or protocol, instead of using "<Resource> Access" as a proper noun. Add "{/*lint ignore messaging*/}" above this line to bypass the linter. (Note that the line number is not accurate if the issue occurs inside a partial.)  messaging     remark-lint